.github/workflows/main.yml

```
@@ -47,40 +47,133 @@ jobs:
           mkdir build
           cd build
           cmake -DCMAKE_C_FLAGS=${{ matrix.platform.flags }} -DCMAKE_CXX_FLAGS=${{ matrix.platform.flags }} -DZYAN_DEV_MODE=ON ${{ matrix.mode.args }} ..
-          
+
       - name: Build
         run: |
           cmake --build build --config ${{ matrix.flavor }}
 
-  build-windows:
-    name: Build ${{ matrix.platform.name }} ${{ matrix.compiler.name }} ${{ matrix.flavor }} (${{ matrix.mode.name }})
+  build-linux-meson:
+    name: Build ${{ matrix.platform.name }} ${{ matrix.compiler.name }} ${{ matrix.flavor }} (${{ matrix.mode.name }}) with meson
```

Let's put the "meson" in the front here: GH UI truncates the action names because they are too long, and I'd rather have this info visible without hovering.

```suggestion
    name: Meson ${{ matrix.platform.name }} ${{ matrix.compiler.name }} ${{ matrix.flavor }} (${{ matrix.mode.name }})
```